packages/input-amount/index.js

```diff
@@ -1,4 +1,4 @@
 export { LionInputAmount } from './src/LionInputAmount.js';
 export { formatAmount } from './src/formatters.js';
-export { parseAmount } from './src/parsers.js';
+export { parseAmount, parseNumber } from './src/parsers.js';
```

Comment:
If we want to align formatNumber/Amount and parseNumber/Amount fully, I think we should put parseNumber inside `localize`.

We can also do it at a later point, but then parseNumber should be private (not exported) inside `input-amount` package